Makefile

```
@@ -14,6 +14,13 @@ install_deps:  ## Install all dependencies.
 setup:  ## Install all dependencies and setup pre-commit
 	make install_deps
 	pre-commit install
+	make .env
+
+.env:  ## Generate .env file
+	@cp .env.example $@
+
+test:  ## Run tests.
```

This now needs a dependency on '.env'
```suggestion
test: .env  ## Run tests.
```